GoogleDataTransport/GDTWatchOSTestApp/Podfile

```diff
@@ -1,3 +1,6 @@
+source 'https://github.com/firebase/SpecsStaging.git'
```

Comment:
This shouldn't be committed, should it?

Reply:
Yes.  It should be in all this repo's Podfiles to enable pre-release testing of dependencies.